### ISSUE TITLE
Tests faild

### ISSUE THREAD

**Issue by mymedia2** [Samsung/rlottie#226]
```
FAILED: test/9f86d08@@vectorTestSuite@exe/test_vpath.cpp.o 
c++ -Itest/9f86d08@@vectorTestSuite@exe -Itest -I../test -Iinc -I../inc -I. -I../ -Isrc/vector -I../src/vector -Isrc/vector/freetype -I../src/vector/freetype -Isrc/vector/pixman -I../src/vector/pixman -Isrc/lottie -I../src/lottie -Isrc/binding/c -I../src/binding/c -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++14 -Os -g '-DDEMO_DIR="/build/rlottie/example/resource/"' -pthread -MD -MQ 'test/9f86d08@@vectorTestSuite@exe/test_vpath.cpp.o' -MF 'test/9f86d08@@vectorTestSuite@exe/test_vpath.cpp.o.d' -o 'test/9f86d08@@vectorTestSuite@exe/test_vpath.cpp.o' -c ../test/test_vpath.cpp
../test/test_vpath.cpp: In member function ‘virtual void VPathTest::SetUp()’:
../test/test_vpath.cpp:8:46: error: converting to ‘const VRectF’ from initializer list would use explicit constructor ‘VRectF::VRectF(int, int, int, int)’
         pathRect.addRect({-10, -20, 100, 100});
                                              ^
../test/test_vpath.cpp:9:58: error: converting to ‘const VRectF’ from initializer list would use explicit constructor ‘VRectF::VRectF(int, int, int, int)’
         pathRoundRect.addRoundRect({0, 0, 100, 100}, 5, 5);
                                                          ^
../test/test_vpath.cpp:10:68: error: converting to ‘const VRectF’ from initializer list would use explicit constructor ‘VRectF::VRectF(int, int, int, int)’
         pathRoundRectZeroCorner.addRoundRect({0, 0, 100, 100}, 0, 0);
                                                                    ^
../test/test_vpath.cpp:11:70: error: converting to ‘const VRectF’ from initializer list would use explicit constructor ‘VRectF::VRectF(int, int, int, int)’
         pathRoundRectHalfCircle.addRoundRect({0, 0, 100, 100}, 60, 60);
                                                                      ^
../test/test_vpath.cpp:12:38: error: converting to ‘const VRectF’ from initializer list would use explicit constructor ‘VRectF::VRectF(int, int, int, int)’
         pathOval.addOval({0,0,100,50});
                                      ^
../test/test_vpath.cpp:13:45: error: converting to ‘const VRectF’ from initializer list would use explicit constructor ‘VRectF::VRectF(int, int, int, int)’
         pathOvalCircle.addOval({0,0,100,100});
                                             ^
```

**Comment by fedya** [comment]
same for me

**Comment by smohantty** [comment]
@mymedia2 , @fedya 
Thanks for reporting.
I have pushed a fix for the same #227 
Also updated CI to build the test suite always #228
